src/core/mdn-annotation.js

```diff
@@ -0,0 +1,174 @@
+import { fetchAndCache } from "./utils";
+import hyperHTML from "hyperhtml";
+import mdnCss from "text!../../assets/mdn-annotation.css";
+
+export const name = "core/mdn-annoatation";
+
+const SPEC_MAP_URL =
+  "https://raw.githubusercontent.com/w3c/mdn-spec-links/master/SPECMAP.json";
+const JSON_BASE = "https://w3c.github.io/mdn-spec-links/";
+const MDN_URL_BASE = "https://developer.mozilla.org/en-US/docs/Web/";
+const MDN_BROWSERS = {
+  // The browser IDs here must match the ones in the imported JSON data.
+  // See the list of browser IDs at https://goo.gl/iDacWP.
+  chrome: "Chrome",
+  chrome_android: "Chrome Android",
+  edge: "Edge",
+  edge_mobile: "Edge Mobile",
+  firefox: "Firefox",
+  firefox_android: "Firefox Android",
+  ie: "Internet Explorer",
+  // nodejs: "Node.js", // no data for features in HTML
+  opera: "Opera",
+  opera_android: "Opera Android",
+  // qq_android: "QQ Browser", // not enough data for features in HTML
+  safari: "Safari",
+  safari_ios: "Safari iOS",
+  samsunginternet_android: "Samsung Internet",
+  // uc_android: "UC browser", // not enough data for features in HTML
+  // uc_chinese_android: "Chinese UC Browser", // not enough data for features in HTML
+  webview_android: "WebView Android",
+};
+
+function fetchAndCacheJson(url, maxAge) {
```

Comment:
Maybe we should remove this function altogether and use `fetchAndCache(...).then(r => r.json())` directly below

Reply:
Since we have other conf like `specMapUrl`, would it be better to group them together within the `run` function? Putting config value and default value together seems more readable to me.

Reply:
Don't have a strong opinion, so ok to keep it in `run()`